app/helpers/moments_helper.rb

```diff
@@ -90,7 +90,7 @@ def moment_form_inputs
         checkboxes: checkboxes_for(@strategies),
         formProps: quick_create_strategy_props
       },
-      moments_viewers_input,
+      get_viewers_input(@viewers, 'moment', 'moments', @moment),
```

Comment:
Are these ivars coming from the controller? If so can we pass the ivars in from the view file as arguments to the method that calls this? [This blog post](http://www.carlosramireziii.com/stop-using-instance-variables-in-partials.html) references using ivars in partials, but I feel the same logic applies to helpers as well